docs/ingestion/tasks.md

```diff
@@ -83,6 +85,70 @@ An example output is shown below, along with a description of the fields:
 }
 ```
 
+### Live report
+
+When a task is running, a live report containing moving average for number of events processed for 1min, 5min, 15min time window can be retrieved at:
```

Comment:
This report also contains information about unparseable events and ingestion state, let's mention that here